docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,129 @@
+## Introduction to Instruments
+
+The Behavioural Database is the part of LORIS that implements the battery of tests 
+and questionnaires which are administered and used to collect data on participants 
+in a given study. These behavioural, psychological, and clinical tests are known as 
+**instruments** or instrument forms.  
+
+Basic instrument forms can be complemented with automated **scoring** and **validation**. 
+The implementation of scoring and validation varies based on the chosen instrument 
+format, the instructions below describe in more details how the implementation should 
+be done for each type.
+
+The **battery** is a list of which instruments are to be administered on each cohort 
+of participants, and typically varies per timepoint and per study site.  For example, 
+the battery of instruments administered at a participant's first visit (e.g. timepoint 
+V01) may depend on which cohort and study site they belong to.  A participant may be 
+tested on a certain set of instruments if they are in cohort A and from site X, but 
+would be tested on different instruments if they are registered in cohort B at site Y. 
+For certain studies, the battery may also depend on the age of the participant. 
+(For more information about Battery assignments, navigate directly to the 
+[Clinical Test Battery](./07_clinical_test_battery.md) section or continue following 
+this order of the).  
+
+### How are Instruments Created?
+
+LORIS instruments can be implemented in one of 2 formats: **LINST** or **PHP**.
+
+**LINST** is a custom LORIS form format which is generated and edited by the LORIS 
+Instrument Builder module. The LINST format allows for the creation of basic forms 
+and can be complemented by a *score* file to allow for automated scoring  and a *rules* 
+file to accommodate for validation of dependencies between form questions and answers.
+
+**PHP** instruments are forms coded manually by a developer in the PHP language. 
+These instruments can also have rules and scoring defined.
+
+Choosing the format in which an instrument is coded is necessary before beginning 
+the process. PHP instruments can eventually be converted to LINST but the opposite is not 
```

Comment:
I don't think this statement is true.. PHP instruments can do arbitrary code which might not be compatible with LINST?

Reply:
valid... and there is no reason to include that anyways